.travis.yml

```diff
@@ -9,8 +9,6 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-install:
-  - ./gradlew assemble
```

Comment:
Those two lines are run to install dependencies before tests:
https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Installation-Step

For Travis CI, Scala projects that don't use sbt behave the same as Java projects. If Travis CI for a Java project detects the gradle wrapper, it will automatically call `./gradlew assemble` during install:
https://docs.travis-ci.com/user/languages/java#Projects-Using-Gradle

I chose to cut those two lines because they were redundant. Hope this helps!